arch/risc-v/src/mpfs/Kconfig

```diff
@@ -773,6 +773,35 @@ config MPFS_CANFD_DATA_BITRATE1
 	default 4000000
 	depends on MPFS_CANFD1
 
+comment "MSS CAN 2.0 Options"
+
+config MPFS_HAVE_MSS_CAN
+	bool "MSS CAN 2.0 peripheral"
+	select ARCH_HAVE_CAN_ERRORS
+	select NET_CAN_EXTID
+	select NET_CAN_HAVE_TX_DEADLINE
+	default n
+
+config MPFS_MSS_CAN0
+	bool "MSS CAN block 0 configured"
+	default n
+	depends on MPFS_HAVE_MSS_CAN
```

Comment:
Here you can just "select MPFS_MSS_CAN". There is no reason to force user to first select "HAVE" and then the block. Just selecting the block can pull in the rest what the MSS_CAN requires